public/views/psychologistList.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MindSet</title>
+    <link rel="stylesheet" href="../styles/reset.css">
+    <link rel="stylesheet" href="../styles/styles.css">
+    <link rel="stylesheet" href="../styles/modal.css">
+    <link rel="stylesheet" href="../styles/psychologist.css">
+    <script async src="../js/modal.js"></script>
+    <script async src="../js/psychologistList.js"></script>
+  </head>
+  <body>
+    <header>
+      <a id="mindSetLogo" href="/"><span>Mind</span>SET</a>
+      <nav>
+        <a href="#">admins</a>
+        <a href="#">applications</a>
+        <a href="#">clients</a>
+        <a href="#">interviews</a>
+        <a href="#">positions</a>
+        <a href="#">Profiles</a>
+        <a id="psychologistNav" href="./psychologistList.html">psychologists</a>
+        <a href="#">sessions</a>
+      </nav>
+    </header>
+    <div class="container">
+      <h2 class="header">Psychologists</h2>
+      <table class="list">
+        <thead>
+          <tr>
+            <th>Frist Name</th>
+            <th>Last Name</th>
+            <th>Username</th>
+            <th>Password</th>
```

Comment:
I don't think displaying a password in a table is a good idea

Reply:
Removed it from the list!